tests/ja4-rules-requires-off/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  min-version: 7.0.3
+  files:
+    - rust/src/ja4.rs
+  script:
+    - grep 'ProvidesFeature(FEATURE_JA4);' src/app-layer-ssl.c > /dev/null
```

Comment:
I do not think this grep is needed, if you have the next one

Reply:
Yeah, right. You wouldn't even have "JA4 support" in the  `--build-info` output if there was no JA4 support in the code, even if is not enabled.